packages/core/status/core-status-server/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export type {
+  ServiceStatusLevel,
+  ServiceStatus,
+  CoreStatus,
+  ServiceStatusLevelId,
+} from '@kbn/core-status-common';
```

Comment:
We usually don't do that, but given the common package is only here because of the common-internal one, I thought it was probably a good idea to re-export the common type though the server package.

Reply:
We'll need to consider the risks of exporting types from other packages. Not a problem now but it may potentially spiral.

Reply:
Yea, I totally agree that we should be very careful about that, and have well defined rules regarding when we allow ourselves to do so.